demo/package.json

```diff
@@ -19,5 +19,6 @@
   },
   "engines": {
     "node": ">=18"
-  }
+  },
+  "packageManager": "pnpm@10.17.1"
```

Comment:
Added packageManager in each packge.json instead of configuring  all the three workflows to set the pnpm version